addons/website_customer/views/website_customer_templates.xml

```diff
@@ -201,6 +221,9 @@
                     <a class="media-heading" t-attf-href="/customers/#{slug(reference)}">
                         <span t-field="reference.self"/>
                     </a>
+                    <t t-if="partner.industry_id">
+                        <span class="label label-default"><t t-esc="partner.industry_id.name"/></span>
```

Comment:
This displays the sector of activity of the Partner, not that of the Reference: `partner.industry_id` -> `reference.industry_id` (and in the `t-if`)